requirements/test_requirements.txt

```diff
@@ -0,0 +1,5 @@
+pytest
+mock
+ovos_utils
+neon_utils
+mycroft
```

Comment:
`mycroft` isn't a valid package, but the module is included in `ovos-core`

Reply:
done